feature_engineering.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+import numpy as np
+#import sys
+#sys.path.insert(1,r"C:\Users\Yakoubi\regression_pipeline")
+from data_preparation import *
+from sklearn.linear_model import Ridge
+from sklearn.feature_selection import SelectFromModel
+
+def feature_selection_ridge(dataset:pd.DataFrame):
+    """
+    Function to select the relevant features to build a model
```

Comment:
- Documenting functions is done this way : https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format, using the reST format
- In a docstring, a sentence begins with an upper case and ends with a point (a point is missing at the end)